gamemodes/terrortown/gamemode/client/cl_vskin/vgui/droleimage_ttt2.lua

```diff
@@ -49,6 +50,20 @@ function PANEL:DoRightClick()
     self:ValueChanged(newValue)
 end
 
+---
+-- @param boolean state
+-- @realm client
+function PANEL:EnableIndicator(state)
+    self.data.indicatorState = state
+end
+
+---
+-- @return boolean
+-- @realm client
+function PANEL:IndicatorEnabled()
+    return self.data.indicatorState or false
+end
```

Comment:
I think indicator is pretty generic and let's stick to the vgui naming scheme for such states.
It seems like the majority uses `SetXY` namings e.g. https://wiki.facepunch.com/gmod/DLabel:SetDark
Let's maybe use `SetIsActiveIndicator` and `GetIsActiveIndicator`

Reply:
Done